frontend/src/views/CellGuide/components/CellGuideCard/connect.ts

```diff
@@ -154,7 +155,7 @@ export function useConnect() {
     setGeneInfoGene,
     cellInfoCellType,
     setCellInfoCellType,
-    organismsList,
+    organismsList: Object.keys(ORGANISM_NAME_TO_TAXON_ID_MAPPING),
```

Comment:
hmm this creates a new object at every render, maybe we should still cache it with `useMemo()` like before?